### PR TITLE
doc: update docs

### PR DESCRIPTION
**Pull request by tinaClin**
- [x] 因為`ButtonGroup` refactor了，所以更新以下documents。 
- modal.mdx
- drawer.mdx
- inputgroup.mdx
- transition.mdx
- pagination.mdx

主要是因為下列changes受到影響
* divider 被拿掉
* selected 被拿掉

- [x] 新增pages/divider.mdx至 styled-ui-docs
- [x] 修正accordion.mdx的icon
- [x] 修正checkbox.mdx的文字

### CODE REVIEW

**Review by austinhung81 — APPROVED** [review]

File: packages/styled-ui-docs/pages/divider.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+```js
+import Divider from '@trendmicro/react-styled-ui/Divider';
+// or
+import { Divider } from '@trendmicro/react-styled-ui';
+```
+
+## Usage
+
+The Divider displays a thin horizontal or vertical line.
```
Comment (cheton) [inline packages/styled-ui-docs/pages/divider.mdx]: The divider renders a thin line vertically or horizontally.
The `Divider` renders a thin line vertically or horizontally.

File: packages/styled-ui-docs/pages/divider.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+```jsx
+<Divider />
+```
+
+### Orientation
+
+Pass the `vertical` prop and set it to either `true` (vertical) or `false` (horizontal).
+
+```jsx
+<Divider vertical={false} />
```
Comment (cheton) [inline packages/styled-ui-docs/pages/divider.mdx:26]: 要不要改用 `orientation` prop ?

https://material-ui.com/components/dividers/#vertical-dividers

```jsx
<Divider orientation="vertical" />
<Divider orientation="horizontal" />
```

File: packages/styled-ui-docs/pages/divider.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Divider
+
+`Divider` is used to visually separate content in a list or group.
```
Comment (cheton) [inline packages/styled-ui-docs/pages/divider.mdx]: The `Divider` is used to visually separate content in a list or group.
The divider is used to visually separate content in a list or group.

File: packages/styled-ui-docs/pages/divider.mdx (hunk 2 above)
Comment (tinaClin) [reply packages/styled-ui-docs/pages/divider.mdx:26]: `ButtonGroup`也有一個相同的prop，需要統一一下嗎？還是各自為政？XDD
image

**Review by roth1002 — APPROVED** [review]

File: packages/styled-ui-docs/pages/divider.mdx (hunk 2 above)
Comment (cheton) [reply packages/styled-ui-docs/pages/divider.mdx:26]: go~

Comment (tinaClin) [reply packages/styled-ui-docs/pages/divider.mdx:26]: 好的~改用`orientation` prop會再開另一個PR做。

**Pull request merged**